modules/client-configuration/pages/clients-alibaba.adoc

```diff
@@ -114,16 +114,16 @@ For more information on activation keys, see xref:client-configuration:activatio
 
 == Register Clients
 
-{alibabaclo} clients are registered in the same way as all other clients.
-
+[[WARNING]]
+====
 Some {alibabaclo}{nbsp}2 instances will fail to register on the first try.
-
 This is due to a to a known bug in the {alibabaclo}{nbsp}2 image.
+The `python-urlgrabber3` package is provided both as a Python pip package and an RPM package, which can cause a conflict on the first attempt to register.
```

Comment:
While I agree with this change, this is not related to this PR.

Please create a separate PR if you want to add this warning.

Reply:
ok, I reverted it.

Background info: The content was already in, I just added the `WARNING` markup.  But it is better to do this separately.